_includes/assets/js/indicatorModel.js

```diff
@@ -377,6 +377,16 @@ var indicatorModel = function (options) {
           }
         }
       },
+      getBackgroundColor = function(datasetIndex) {
```

Comment:
Maybe you could change this to `getBackground()` now that it does a bit more than setting a background _color_ (it sets a pattern as well).

Also, (just a suggestion) consider splitting your function into the part of changing color and the part of changing pattern (solid or diagonal):

```javascript
function getBackground(datasetIndex){
    var color = getBackgroundColor(datasetIndex);
    var background = getBackgroundPattern(datasetIndex, color);
    return background;
}
```

Reply:
Good ideas, I made these changes.